Source/Core/LMCConst.hh

```diff
@@ -28,33 +28,33 @@ public:
 
     constexpr static double Q()
     {
-        return 1.60217653E-19;
+        return 1.602176634E-19;  // updated from https://pdg.lbl.gov/2021
     } //!< elementary charge  in C(>0)
 
     constexpr static double Hbar()
     {
-        return 1.05457168E-34;
+        return 1.054571817E-34; // updated from https://pdg.lbl.gov/2021
     }//!< hbar in J s-1
 
     constexpr static double HbarC_eV()
     {
-        return 197.326968E-9;
+    	return 197.3269804; // updated from https://pdg.lbl.gov/2021
```

Comment:
if units should be in eV * m then a factor 1e-9 is missing. In PDG they are given as MeV (1e6) * fm (1e-15) thus canceling the 1e6 and 1e-15 results in 1e-9.